tests/operators/test_local.py

```diff
@@ -567,8 +648,10 @@ def test_run_operator_dataset_emission_is_skipped(caplog):
     assert run_operator.outlets == []
 
 
-# TODO: Make test compatible with Airflow 3.0. Issue:https://github.com/astronomer/astronomer-cosmos/issues/1713
-@pytest.mark.skipif(version.parse(airflow_version).major == 3, reason="Test need to be updated for Airflow 3.0")
+@pytest.mark.skipif(
+    version.parse(airflow_version).major == 3,
+    reason="Airflow 3.0 only supports assets when setting enable_dataset_alias=True",
```

Comment:
I guess the skip reason needs to be updated for this test. Looks like the reason is a copy of the above test.